Dynatrace/dynatrace-operator
# Collect ConfigMaps in operator namespace into support archive.

## Description

Collect all ConfigMaps that exist in the operator namespace into the support archive.

## How can this be tested?

Run `support-archive` command and check if all `ConfigMaps` existing in your operator namespace are part of the support archive zip.

## Checklist
